tests/components/keymitt_ble/test_config_flow.py

```diff
@@ -0,0 +1,173 @@
+"""Test the MicroBot config flow."""
+
+from unittest.mock import ANY, patch
+
+from homeassistant.config_entries import SOURCE_BLUETOOTH, SOURCE_USER
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_ADDRESS
+from homeassistant.data_entry_flow import FlowResultType
+
+from . import (
+    SERVICE_INFO,
+    USER_INPUT,
+    MockMicroBotApiClient,
+    MockMicroBotApiClientFail,
+    patch_async_setup_entry,
+)
+
+from tests.common import MockConfigEntry
+
+DOMAIN = "keymitt_ble"
+
+
+async def test_bluetooth_discovery(hass):
+    """Test discovery via bluetooth with a valid device."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={"source": SOURCE_BLUETOOTH},
+        data=SERVICE_INFO,
+    )
+    assert result["type"] == FlowResultType.FORM
+    assert result["step_id"] == "init"
+
+    with patch_async_setup_entry() as mock_setup_entry:
+        result = await hass.config_entries.flow.async_configure(
+            result["flow_id"],
+            USER_INPUT,
+        )
+        await hass.async_block_till_done()
+
+    assert result["type"] == FlowResultType.FORM
```

Comment:
Please continue all config flow tests to either a create entry, error or abort result.